api-java-mixin.raml

```diff
@@ -260,6 +450,82 @@ types:
       ProductUpdateAction, ProductUpdateBuilder>'
   ProductUpdateAction:
     (java-extends): 'com.commercetools.api.models.ResourceUpdateAction<ProductUpdateAction>'
+  ProductSetKeyAction:
+    (java-mixin): |
+      public static ProductSetKeyAction ofUnset() {
+          return ProductSetKeyAction.of();
+      }
+  ProductSetImageLabelAction:
+    (java-mixin): |
+      public static ProductSetImageLabelAction ofUnset(final Long variantId, final String imageUrl, @javax.annotation.Nullable final Boolean staged) {
+          return ProductSetImageLabelActionBuilder.of().variantId(variantId).imageUrl(imageUrl).staged(staged).build();
+      }
+
+      public static ProductSetImageLabelAction ofUnsetStaged(final Long variantId, final String imageUrl) {
+          return ProductSetImageLabelActionBuilder.of().variantId(variantId).imageUrl(imageUrl).build();
+      }
+  ProductSetTaxCategoryAction:
+    (java-mixin): |
+      public static ProductSetTaxCategoryAction ofUnset() {
+          return ProductSetTaxCategoryAction.of();
+      }
+  ProductSetCustomerAction:
+    (java-mixin): |
+      public static ProductSetCustomerAction ofUnset() {
+          return ProductSetCustomerAction.of();
+      }
+  ProductSetProductPriceCustomFieldAction:
+    (java-mixin): |
+      public static ProductSetProductPriceCustomFieldAction ofUnset(final String name, final String priceId) {
+          return ProductSetProductPriceCustomFieldActionBuilder.of().name(name).priceId(priceId).build();
+      }
+
+      public static ProductSetProductPriceCustomFieldAction ofUnsetStaged(final String name, final String priceId,
+              @javax.annotation.Nullable final Boolean staged) {
+          return ProductSetProductPriceCustomFieldActionBuilder.of().name(name).priceId(priceId).staged(staged).build();
+      }
+  ProductSetAttributeAction:
+    (java-mixin): |
+      public static ProductSetAttributeAction ofVariantId(final Long variantId, final String name, @javax.annotation.Nullable final com.fasterxml.jackson.databind.JsonNode value, @javax.annotation.Nullable final Boolean staged) {
+          return ProductSetAttributeActionBuilder.of().variantId(variantId).staged(staged).build();
```

Comment:
```suggestion
      public static ProductSetAttributeAction ofVariantId(final Long variantId, final String name, final com.fasterxml.jackson.databind.JsonNode value, final Boolean staged) {
          return ProductSetAttributeActionBuilder.of().variantId(id).name(name).value(value).staged(staged).build();
```

Reply:
I think you meant `.variantId(variantId).name(name).value(value)`

Reply:
yeah :)